hummingbot/market/coinbase_pro/coinbase_pro_market.pyx

```diff
@@ -879,9 +896,14 @@ cdef class CoinbaseProMarket(MarketBase):
                     if type(client_order_id) is str:
                         order_id_set.remove(client_order_id)
                         successful_cancellations.append(CancellationResult(client_order_id, True))
-        except Exception:
+                    else:
+                        self.logger().info(
+                            f"failed to cancel order with error: "
```

Comment:
You are using a different log level for this in liquid and coinbase. Could you use the same level ?. Also not sure if logging the type of client order id is needed here.

Reply:
Ready to push the other changes but I think logging the type of client_order_id is useful here only when it is an Exception. I'm going to change it to repr(client_order_id) so that if it is an Exception it displays the Exception class as well as the contents.